packages/vuetify/src/composables/teleport.ts

```diff
@@ -0,0 +1,32 @@
+import { computed, warn } from 'vue'
+import type { Ref } from 'vue'
+
+export function useTeleport (target: Ref<boolean | string | Element>) {
+  const teleportTarget = computed(() => {
+    const _target = target.value
+
+    if (_target === true) return undefined
+
+    const targetElement =
+      _target === false ? document.body
+      : typeof _target === 'string' ? document.querySelector(_target)
+      : _target
+
+    if (targetElement == null) {
+      warn(`Unable to locate target ${_target}`)
```

Comment:
Are we using this or our own warn method?

Reply:
Idk, this has the component stack trace but says Vue instead of vuetify

Reply:
This is something we can add for beta testing in July